.setup/install_worker.sh

```diff
@@ -12,17 +12,17 @@ SUPERVISOR_USER=submitty
 echo "checking ${SUPERVISOR_USER} user"
 # Create the submitty user here on the worker machine
 # This is the user that submitty_daemon on the main vagrant machine will ssh into.
-if ! cut -d ':' -f 1 /etc/passwd | grep -q ${SUPERVISOR_USER} ; then
+if ! cut -d ':' -f 1 /etc/passwd | grep -q "${SUPERVISOR_USER}" ; then
     echo "attempting to add ${SUPERVISOR_USER} user"
     #set up submitty user with password 'submitty'
     useradd -m -p $(openssl passwd -crypt submitty) -c "First Last,RoomNumber,WorkPhone,HomePhone" "${SUPERVISOR_USER}"
```

Comment:
From ShellCheck:
> [SC2046](https://www.shellcheck.net/wiki/SC2046): Quote this to prevent word splitting.

```suggestion
    useradd -m -p "$(openssl passwd -crypt submitty)" -c "First Last,RoomNumber,WorkPhone,HomePhone" "${SUPERVISOR_USER}"
```